### ISSUE TITLE
[BOOM-5562] An empty () is displayed in the product page FO

### ISSUE THREAD

**Issue by prestonBot** [PrestaShop/PrestaShop#9642]
> This issue has been migrated from the Forge. Read the [original ticket here](http://forge.prestashop.com/browse/BOOM-5562).

- _**Reporter:**_ khouloud.belguith
- _**Created at:**_ Thu, 17 May 2018 17:28:21 +0200

Hi,

Creating a product with a price per unit and creating a specific price by applying a discount >price of the product, an empty () will be displayed on the product page FO.

 <span class="image-wrap" style=""><a id="67830_thumb" href="http://forge.prestashop.com/secure/attachment/67830/67830_image-2018-05-17-17-23-48-236.png" title="image-2018-05-17-17-23-48-236.png" file-preview-type="image" file-preview-id="67830" file-preview-title="image-2018-05-17-17-23-48-236.png"><img src="http://forge.prestashop.com/secure/thumbnail/67830/_thumb_67830.png" style="border: 0px solid black" /></a></span>

Best regards, Khouloud

---
#### Describe the bug

Creating a product with a price per unit and creating a specific price by applying a discount >price of the product, an empty () will be displayed on the product page FO.

#### Expected behavior

No display of **`()`**

#### Steps to Reproduce

1. **BO**>**SELL**>**Catalog**>**Products**
2. Create a product with price (Tax included) = 10Euro / Price per unit (tax excl.) = 4Euro/Kg
3. Create a specific price =&gt; Apply a discount of 12Euro (Tax included)
4. Navigate to product page FO
5. See error => An empty () is displayed in the product page FO

**Screenshots**

![image](https://user-images.githubusercontent.com/23308427/106259767-f2af1200-621f-11eb-8150-5aa6d6989a97.png)

![image](https://user-images.githubusercontent.com/23308427/106259787-f773c600-621f-11eb-9e2c-f42a4988845c.png)

#### Additional information
* PrestaShop version: 1.7.7.1
* PHP version: 7.2

**Comment by prestonBot** [comment]
> This comment has been migrated from the Forge. Read the [original comment here](http://forge.prestashop.com/browse/BOOM-5562#comment-173330).

- _**Author:**_ marion_francois
- _**Created at:**_ Thu, 2 Aug 2018 14:42:29 +0200

<p><a href="http://forge.prestashop.com/secure/ViewProfile.jspa?name=rntie" class="user-hover" rel="rntie">Reddy Ntie </a> Can you try to reproduce on the last version ?</p>

**Comment by colinegin** [comment]
Hello @khouloudbelguith do you reproduce this issue ? Thanks !

**Comment by khouloudbelguith** [comment]
Hi @colinegin,

Yes!

I have the same issue with PS1.7.6.2
![image](https://user-images.githubusercontent.com/16067358/71086241-a62e8180-2199-11ea-89e6-5611f00c3b62.png)

Thanks!

**Comment by PrestaEdit** [comment]
Seems to be fixed onto 8.0.0-beta.1

**Comment by Hlavtox** [comment]
@hibatallahAouadni Can you recheck and close if fixed?

**Comment by HanaRebaiQA** [comment]
Hello everybody, 

I tried to reproduce this issue with PS1788 and PS8.0.x and here is my feedback : 

- I was able to reproduce the issue on **PS1788 : NOK** :x: 

![image](https://user-images.githubusercontent.com/92912932/208942041-8441a369-4864-4c7d-9c00-b644fce964c9.png)

https://user-images.githubusercontent.com/92912932/208942051-39474542-3204-4c72-affc-d50fa738adbe.mp4


- But, I was unable to reproduce it on **PS8.0.x : OK** :heavy_check_mark: 

![image](https://user-images.githubusercontent.com/92912932/208941964-8340e30a-da51-4210-a5d3-b31ba6cd8bc8.png)

https://user-images.githubusercontent.com/92912932/208942007-4da31a5b-01ae-4e43-b9ba-c33b133e5cd4.mp4


Since for now, only security and critical issues will be fixed on 1.7.8.x, we can consider that this issue is laready fixed on 8.0.x. 

So, I'll close this issue.

Thank You!
